client/src/App.js

```diff
@@ -1,14 +1,29 @@
 import React from "react";
-//import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import GlobalStyles from "./GlobalStyles";
 import EventPage from "./pages/EventPage";
+import SearchPage from "./pages/SearchPage";
+import styled from "@emotion/styled";
+
+const AppContainer = styled.div`
+  margin-bottom: 100px;
```

Comment:
Can you adapt this to REM or %?
When I unclick it, there seems to be no effect: 
![image](https://user-images.githubusercontent.com/68224104/95450800-1dbae080-0967-11eb-9f10-6327646b581a.png)

Reply:
still have to warm up with rem, thx man 👍